src/components/UI/ExternalLink/ExternalLink.tsx

```diff
@@ -25,7 +26,8 @@ export const ExternalLinkEtherscanAddress: FunctionComponent<ExternalLinkEthersc
   address,
   ...props
 }) => {
-  const href = makeEtherscanAddressURL(address);
+  const { currentChainId } = useProviderContext();
+  const href = currentChainId ? makeEtherscanAddressURL(address, currentChainId) : "#";
 
```

Comment:
would the `#` scenario external link to nowhere?

Reply:
Yea it will not go to anywhere when it's `#` which is when the network is not supported.